pkg/config/config_test.go

```diff
@@ -15,120 +15,169 @@
 package config
 
 import (
-	"fmt"
 	"io/ioutil"
+	"os"
 	"testing"
 
-	"github.com/sealerio/sealer/common"
+	"github.com/stretchr/testify/assert"
 
 	v1 "github.com/sealerio/sealer/types/api/v1"
 )
 
-/* (t *testing.T) {
-	type fields struct {
-		configs     []v1.Config
-		clusterName string
-	}
-	type args struct {
-		clusterfile string
-	}
-	tests := []struct {
-		name    string
-		fields  fields
-		args    args
-		wantErr bool
-	}{
-		{
-			"test dump clusterfile configs",
-			fields{
-				configs:     nil,
-				clusterName: "my-cluster",
-			},
-			args{clusterfile: "test/test_clusterfile.yaml"},
-			false,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			c := &Dumper{
-				Configs: tt.fields.configs,
-				Cluster: &v2.Cluster{},
-			}
-			c.Cluster.Name = tt.fields.clusterName
-			configs, err := utils.DecodeCRDFromFile(tt.args.clusterfile, common.Config)
-			if err != nil {
-				t.Error(err)
-				return
-			}
-			if err := c.Dump(configs.([]v1.Config)); (err != nil) != tt.wantErr {
-				t.Errorf("Dump() error = %v, wantErr %v", err, tt.wantErr)
-			}
-		})
+func Test_getMergeConfig(t *testing.T) {
+	testFileData := `apiVersion: v1
+data:
+  key1: myConfigMap1
+kind: ConfigMap
+metadata:
+  name: myConfigMap1
+---
+apiVersion: v1
+data:
+  key2: myConfigMap2
+kind: ConfigMap
+metadata:
+  name: myConfigMap2
+---
+apiVersion: v1
+data:
+  key3: myConfigMap3
+kind: ConfigMap
+metadata:
+  name: myConfigMap3
+`
+
+	wantedFileData := `apiVersion: v1
```

Comment:
Put the "apiVersion: v1" to the next line? It will look better.

Reply:
yes，I also wrote this before, but it need to compare the results bytes , this will add an extra byte `\n`.